porch/pkg/apiserver/apiserver.go

```diff
@@ -147,6 +148,13 @@ func (c completedConfig) getCoreClient() (client.WithWatch, error) {
 	if err := configapi.AddToScheme(scheme); err != nil {
 		return nil, fmt.Errorf("error building scheme: %w", err)
 	}
+
+	// We query the apiserver, even if we could query directly; this will then work with CRDs etc.
+	// TODO: We need to think about priority-and-fairness with loopback queries
```

Comment:
This comments seems out of place. Registering porchapi to current scheme doesn't involve querying apiserver. 
Perhaps, misplaced ?

Reply:
This is where we enable it, but fair point - I moved to the callsite!